src/modules/auth/LoginForm/LoginForm.tsx

```diff
@@ -87,7 +87,10 @@ export default function LoginForm({
             <Button type="submit" isLoading={isLoading}>
               {t('login.button')}
             </Button>
-            <Link href="/" textAlign="center">
+            <Link
+              href="http://localhost:3000/forget-password"
```

Comment:
You don't need baseUrl here, and you can also make this a constant

Reply:
this has been updated